### ISSUE TITLE
Coder CLI asking for input on immutable parameters, resulting in start/restart from CLI being broken

### ISSUE THREAD

**Issue by mafredri** [coder/coder#9171]
I'm unable to start my `dev.coder.com` workspace named `mafredri/w`. When I type `coder start w`, I see the following:

```
❯ coder start w

  > Pittsburgh
    Helsinki
    Sydney
    São Paulo
```

If I select one, the result is an error:

```
Parameter "Region" is not mutable, so it can't be updated after creating a workspace.
```

This affects both v2.0.2 and v2.1.0 versions of the CLI (none other tested).

Start via web works fine.

**Comment by mtojek** [comment]
Taking a look

**Comment by mafredri** [comment]
I'm still seeing this with https://github.com/coder/coder/releases/tag/v2.1.1

<img width="649" alt="image" src="https://github.com/coder/coder/assets/147409/2d9c9697-82a4-4317-ae61-cd50ce03e09c">

Re-opening.

**Comment by mtojek** [comment]
I have the fix included [here](https://github.com/coder/coder/pull/9247), but I can extract it to a separate PR to unblock you.

**Comment by mafredri** [comment]
Thanks @mtojek, but no hurry on my part, I can get around it via the WebUI.
